client/src/main/java/org/spine3/type/TypeUrl.java

```diff
@@ -38,10 +39,13 @@
 import org.spine3.envelope.MessageEnvelope;
 import org.spine3.type.error.UnknownTypeException;
 
+import java.util.List;
 import java.util.Objects;
-import java.util.regex.Pattern;
 
+import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.protobuf.Internal.getDefaultInstance;
```

Comment:
I'm glad you spotted. I specifically put it into the `import static` section to highlight the fact. I am not aware of any means that can give us the features we need from that `Internal` class.